src/tiled/objectselectiontool.cpp

```diff
@@ -1020,7 +1020,8 @@ void ObjectSelectionTool::updateMovingItems(const QPointF &pos,
         mapObject->setPosition(newPos);
     }
 
-    emit mapDocument()->mapObjectModel()->objectsChanged(changingObjects());
+    mapDocument()->mapObjectModel()->emitObjectsChanged(changingObjects(),
+                                                        QList<MapObjectModel::Columns>() << MapObjectModel::Position);
```

Comment:
So I see now that currently the only interesting column to be included here is the Position column. I think a helper override taking a single `MapObjectModel::Column` argument, which can in turn just call the list version, would be nice.